apps/web/app/locales/pl/translation.json

```
@@ -929,7 +931,12 @@
       "maxTitleLengthExceeded_few": "Tytuł nie może mieć więcej niż {{count}} znaki",
       "maxTitleLengthExceeded_many": "Tytuł nie może mieć więcej niż {{count}} znaków",
       "maxTitleLengthExceeded_other": "Tytuł nie może mieć więcej niż {{count}} znaków",
-      "aiMentorAvatarIncorrectType": "Nieprawidłowy typ pliku. Prześlij obraz w formacie JPG, JPEG, PNG lub SVG."
+      "aiMentorAvatarIncorrectType": "Nieprawidłowy typ pliku. Prześlij obraz w formacie JPG, JPEG, PNG lub SVG.",
+      "languageNotSupported": "Ten kurs nie obsługuje tego języka.",
+      "updateCourseMissingLanguage": "Nie podano języka do tłumaczenia",
+      "successfullyDeletedLanguage": "Język został pomyślnie usunięty z kursu",
+      "invalidLanguageToDelete": "Nie możesz usunąć tego języka",
+      "cannotModifyQuestionsInNonBaseLanguage": "Nie można modyfikować pytań w języku innym niż domyślny."
```

```suggestion
      "cannotModifyQuestionsInNonBaseLanguage": "Nie można modyfikować pytań w języku innym niż bazowy."
```
W innych miejscach tak określamy ten język.